haproxy_exporter.go

```diff
@@ -325,7 +329,7 @@ loop:
 			continue loop
 		default:
 			log.Errorf("Unexpected error while reading CSV: %v", err)
-			e.csvParseFailures.Inc()
+			e.up.Set(0)
```

Comment:
@grobie without this unix socket read errors would be marked as csv parse failures, which might or might not be what you want. Up to you.